deployer.go

```diff
@@ -121,9 +121,7 @@ func (d *deployer) deployUnit(wantedUnit *schema.Unit) error {
 
 	wuf := schema.MapSchemaUnitOptionsToUnitFile(wantedUnit.Options)
 	cuf := schema.MapSchemaUnitOptionsToUnitFile(currentUnit.Options)
-	wufUnescapedString := strings.Replace(wuf.String(), "\\\n", "", -1)
-	if strings.Replace(wufUnescapedString, " ", "", -1) != strings.Replace(cuf.String(), " ", "", -1) {
-
+	if wuf.Hash() != cuf.Hash() {
```

Comment:
func (u *UnitFile) Hash() Hash {
	return Hash(sha1.Sum(u.Bytes()))
}

This won't work if there are extra " ", "\n", "\" in there as the hash will be different.